tools/amazon_agent_settings/agent_gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
```

Comment:
This file doesn't belong in tools...not sure where...maybe something else, like scripts?  Or perhaps this belongs with the amazon gem?

Reply:
@Fryguy I don't think this belongs with the gem, because it is being used to *load* the amazon gem (among other things) by the code that is being deployed out to the Amazon instance to run SSA.

Reply:
It's just a file and it's being read by the worker in order to pass it via userdata, so technically it can live anywhere, and the best place IMO is the plugin gem.